src/blog/contributors-youve-earned-a-gitpoap/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Hey Trufflesuite contributors, you’ve earned a POAP!
+hide:
+  - navigation
+---
+
+**By Kingsley Arinze**
+
+Hey Trufflesuite contributors, we’re excited to share that in collaboration with the [GitPOAP](https://gitpoap.io/) team, we will be issuing POAPs in appreciation of your awesome contributions over the years. 
+
+## What are POAPs?
+
+POAPs (Proof Of Attendance Protocol) are badges issued as NFTs (non-fungible token) to prove that the recipient/owner has carried out an action or has attended an event. They are like gold medals from the Olympics, except these exist as digital collectibles. 
+
+## Who's eligible to claim a GitPOAP?
+
+Great question. Annual contributor POAPs will be issued to those who have contributed at least once all year since the start of the following projects:
+
+- Ganache
+- Truffle
+- trufflesuite.com website
+
+Those of you who have contributed to all three projects can claim all three POAPs! 
+
+Here are some POAPs you should expect to receive as a TruffleSuite contributor.
+
+![gitpoap examples](./poap-examples.png)
```

Comment:
This alt description is not descriptive enough. If the person was blind, how would you describe the contents?

Reply:
great point 🙏 updated to "_3 examples of GitPOAPs you could expect to receive as a contributor_" 1f372370